src/api/handlers/PetFindManyHandler.ts

```diff
@@ -1,29 +1,21 @@
-import { Handler } from "aws-lambda";
+import { Handler, ProxyResult } from "aws-lambda";
+import { ProxyResultBuilder } from "../core/ProxyResultBuilder";
 import { PetService } from "../services/PetService";
 
 class PetFindManyHandler {
-    private service: PetService;
+    private petService: PetService;
 
-    constructor() {
-        this.service = new PetService();
+    public initializeDependencies(): void {
```

Comment:
@Aristevao here you can still using the `constructor` instead add this `initializeDependencies()` function.